### DISCUSSION TITLE
`has-pull-request` label to identify an issue which already has pull request

### DISCUSSION THREAD

**Discussion by MrJithil** [ethereum/ethereum-org-website#9311] [Ideas]
I am actively looking for contributing to this repo. To identify an issue which nobody started working on it and free to take by anyone should be identified by a label.

Some label like `has-pull-request` on issues would be a meaningful to contributors.

What's your thoughts?

**Comment by corwintines** [comment]
Im not sure its necessary honestly. If you link the issue in the PR you get an icon beside the issue already
![Screen Shot 2023-01-26 at 8 41 24 AM](https://user-images.githubusercontent.com/15589226/214912744-3bc84e52-1108-42ea-b903-6c24209e8081.png)

**Reply by MrJithil** [reply]
This icon will not show always.

**Comment by MrJithil** [comment]
Discussion can be closed.
